hw/ip/flash_ctrl/data/flash_ctrl.hjson.tpl

```diff
@@ -527,7 +527,7 @@
           },
           { bits: "27:16",
             name: "NUM",
-            desc: "Number of bus words the flash operation should read or program.",
+            desc: "One fewer than the number of bus words the flash operation should read or program.",
```

Comment:
thanks! could you add an example just in case?
like..if you intend to program 10, write 9 here.

Reply:
Done :)